python_modules/dagster-graphql/dagster_graphql/schema/roots/query.py

```diff
@@ -535,7 +535,7 @@ def resolve_assetsLatestInfo(self, graphene_info, **kwargs):
         step_keys_by_asset: Dict[AssetKey, List[str]] = {
             node.external_asset_node.asset_key: node.external_asset_node.op_names
             for node in results
-            if not asset_keys or node.assetKey in asset_keys
+            if node.assetKey in asset_keys
```

Comment:
Oh nice! Man, It drives me nuts that empty Javascript arrays are truthy, but it's also a bit weird that empty python arrays are falsey...

I like that we're removing the ability to get this data instance-wide (eg: omitting asset keys), it feels like a good practice for resolvers that are expensive like this.

Could we also change the argument to be required? Right now it looks like the generated GraphQL schema still thinks `assetKeys` is optional.

I think 
```
assetKeys=graphene.Argument(graphene.List(graphene.NonNull(GrapheneAssetKeyInput))),
```

Should be:

```
assetKeys=graphene.Argument(non_null_list(GrapheneAssetKeyInput))
```

Reply:
Thanks for the feedback! Yep, I've updated this argument to be required.